src/views/ProductView.vue

```diff
@@ -275,7 +294,15 @@
                     <v-divider></v-divider>
                     <v-card-text>
                         <v-container>
-                            <v-row align="start" no-gutters>
+                            <v-row align="start" dense>
+                                <v-col>Weight:</v-col>
+                                <v-col>{{
+                                    productVariant?.currentVersion.weight +
+                                    ' ' +
+                                    unitUsedForWeightInformation
```

Comment:
imho this does not work
the backend requires this stuff to be saved in kg
if you want to support others, you need to do unit conversion yourself

Reply:
Oh, okay. Didn't know its kg in the backend. I will change it to kg only.